tests/conftest.py

```diff
@@ -15,3 +24,45 @@ def vcr_config():
     return {
         "filter_headers": [CROSSREF_HEADER_KEY, SEMANTIC_SCHOLAR_HEADER_KEY],
     }
+
+
+@pytest.fixture
+def tmp_path_cleanup(tmp_path: Path) -> Generator[Path, None, None]:
```

Comment:
```suggestion
def tmp_path_cleanup(tmp_path: str | os.PathLike) -> Iterator[str | os.PathLike]:
```

Let's stay general if we can

Reply:
the `.exists` method only exists on a Path object.